inc/options/minify.php

```diff
@@ -592,8 +930,58 @@
 			<tr>
 				<th colspan="2">
 					<ul>
-						<li><?php _e( 'Enable <acronym title="Hypertext Transfer Protocol">HTTP</acronym> compression in the "Cascading Style Sheets &amp; JavaScript" section on <a href="admin.php?page=w3tc_browsercache">Browser Cache</a> Settings tab.', 'w3-total-cache' ); ?></li>
-						<li><?php _e( 'The <acronym title="Time to Live">TTL</acronym> of page cache files is set via the "Expires header lifetime" field in the "Cascading Style Sheets &amp; JavaScript" section on <a href="admin.php?page=w3tc_browsercache">Browser Cache</a> Settings tab.', 'w3-total-cache' ); ?></li>
+						<li>
+							<?php
+							echo wp_kses(
+								sprintf(
+									// translators: 1 opening HTML acronym tag, 2 closing HTML acronym tag,
+									// translators: 3 opening HTML a tag to W3TC Browsercache admin page, 4 closing HTML a tag.
+									__(
+										'Enable %1$sHTTP%2$s compression in the "Cascading Style Sheets &amp; JavaScript" section on %3$sBrowser Cache%4$s Settings tab.',
+										'w3-total-cache'
+									),
+									'<acronym title="' . __( 'Hypertext Transfer Protocol', 'w3-total-cache' ) . '">',
+									'</acronym>',
+									'<a href="admin.php?page=w3tc_browsercache">',
```

Comment:
Use `esc_url()` and `admin_url()` for the `href`.

Reply:
Added esc_url and admin_url to href